src/Host/Client/Impl/DataInspection/IRActiveBindingInfo.cs

```diff
@@ -6,5 +6,9 @@ namespace Microsoft.R.DataInspection {
     /// Describes the result of evaluating an expression that references an active binding. 
     /// </summary>
     public interface IRActiveBindingInfo : IREvaluationResultInfo {
+        /// <summary>
+        /// Value of the active binding after evaluation.
+        /// </summary>
+        IRValueInfo Value { get; }
```

Comment:
Also, there is a mismatch here between the name of the property, and the name of the corresponding JSON entry. Ideally these should be the same, except for the naming convention (I think the only other place where we deviate currently is "type" vs "TypeName"). I'd call it something like `ComputedValue` for clarity.